src/mobility.ts

```diff
@@ -45,7 +45,7 @@ export const MobilityOperator = z.object({
   benefits: z.array(OperatorBenefit).optional().default([]),
   brandAssets: z
     .object({
-      brandLastModified: z.date(),
+      brandLastModified: z.string().date(),
```

Comment:
This _looks_ like a breaking change, but maybe more correct considering what's in firestore-configuration. Could you upgrade config-specs in the app as well after this is merged, so we know it doesn't create issues there?

Reply:
Yeah I thought so too, but if you check https://github.com/AtB-AS/config-specs/pull/93/files it was z.string().date() before, and the json schema translates it into `type:"string"` without the `format:"date"`, so I think this should be safe. But I'll test the config specs in app after this is merged :)